packages/icons/src/ui-icon.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { ColorCategories, TextSize, ThemeContext, getTextSize, getThemeStyling } from '@uireact/foundation';
+
+import { UiIconProps, privateIconProps } from './types';
+import { getDynamicMapper } from './theme';
+
+import click from './svgs/click.svg';
+import picture from './svgs/040-picture.svg';
+import rightArrow from './svgs/right-arrow.svg';
+
+const Span = styled.span<privateIconProps>`
+  ${(props) => `
+    ${getThemeStyling(
+      props.customTheme,
+      props.selectedTheme,
+      getDynamicMapper(props.category || ColorCategories.fonts)
+    )}
+
+    img {
+      width: ${getTextSize(props.customTheme, props.size || TextSize.small)};
+    }
+  `}
+`;
+
+const getIcon = (id: string): string => {
```

Comment:
Will think in a better solution for this... I'm not quite sure yet about this logic and how to export all the different icons we will support at some point.